maintainers/maintainer-list.nix

```diff
@@ -5412,6 +5412,12 @@
     githubId = 5047140;
     name = "Victor Collod";
   };
+  mveytsman = {
```

Comment:
You also need to make sure your PR eventually has 2 commits:

```
maintainers: add mveytsman
htmlbeautifier: init at 1.3.1
```

Reply:
Thanks, that wasn't clear to me, I amended the history accordingly